build.gradle

```diff
@@ -182,51 +188,74 @@ subprojects {
         delete "$buildDir/libs"
     }
 
-    shadowJar {
-        configurations = [project.configurations.shade]
-        relocate 'com.tterrag.registrate', 'com.enderio.repack.registrate'
-    }
-
-    reobf {
-        shadowJar {}
-    }
+    if (project.name != "endercore") {

```

Comment:
Can we not just put it in base? All the other mods depend on this anyway

Reply:
I have tried this, but we wind up with very odd obfuscation problems. Registrate is obfuscated while base is not, meaning some methods become unavailable. I'm up for finding another solution but after numerous attempts I couldn't get this working nicely.